packages/api-client/src/views/Request/RequestSection/RequestTable.vue

```diff
@@ -175,26 +176,26 @@ const flattenValue = (item: RequestExampleParameter) => {
       </DataTableCell>
       <DataTableCell
         v-if="showUploadButton"
-        class="group/upload p-1 overflow-hidden relative text-ellipsis whitespace-nowrap">
+        class="group/upload relative overflow-hidden text-ellipsis whitespace-nowrap p-1">
         <template v-if="item.file">
           <div
-            class="text-c-2 max-w-[100%] overflow-hidden filemask flex items-end justify-end">
+            class="text-c-2 filemask flex max-w-[100%] items-end justify-end overflow-hidden">
             <span>{{ item.file?.name }}</span>
           </div>
           <button
-            class="absolute bg-b-2 font-medium centered-x centered-y hidden rounded text-center p-0.5 w-[calc(100%_-_8px)] group-hover/upload:block text-xs"
+            class="bg-b-2 centered-x centered-y absolute hidden w-[calc(100%_-_8px)] rounded p-0.5 text-center text-xs font-medium group-hover/upload:block"
             type="button"
             @click="emit('removeFile', idx)">
             Delete
           </button>
         </template>
         <template v-else>
           <ScalarButton
-            class="bg-b-2 hover:bg-b-3 border-0 py-px text-c-2 shadow-none"
+            class="bg-b-2 hover:bg-b-3 text-c-2 border-0 py-px shadow-none"
             size="sm"
             variant="outlined"
             @click="handleFileUpload(idx)">
-            <span>File</span>
+            <span>Upload File</span>
```

Comment:
to be precise, you just *select* a file here, the upload only happens when you send a request

```suggestion
            <span>Select File</span>
```

Reply:
but you are telling it to upload a file 😉 @cameronrohani what say you

Reply:
for now 😏 <br>— as we already use that wording + give the action context - i guess matter of taste

Reply:
but the *button* doesn’t upload, a button label should describe what the button does 😶

Reply:
tomato potato I say lol. Normally I would agree with you hans but in this case, even though its technically wrong, it feels right 😉